examples/30_extended/task_manual_iteration_tutorial.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tasks: retrieving splits
+========================
+
+Tasks define a target and a train/test split. Normally, they are the input to the function
+``openml.runs.run_model_on_task`` which automatically runs the model on all splits of the task.
+However, sometimes it is necessary to manually split a dataset to perform experiments outside of
+the functions provided by OpenML. One such example is in the benchmark library
+`HPOlib2 <https://github.com/automl/hpolib2>`_ which extensively uses data from OpenML,
+but not OpenML's functionality to conduct runs.
+"""
+
+import openml
+
+####################################################################################################
+task_id = 233
+task = openml.tasks.get_task(task_id)
+
+####################################################################################################
+# Now that we have a task object we can obtain the number of repetitions, folds and samples as
+# defined by the task:
+
+n_repeats, n_folds, n_samples = task.get_split_dimensions()
+
+####################################################################################################
+# * ``n_repeats``: Number of times the model quality estimation is performed
+# * ``n_folds``: Number of folds per repeat
+# * ``n_samples``: How many data points to use. This is only relevant for learning curve tasks
+#
+# A list of all available estimation procedures is available
+# `here <https://www.openml.org/search?q=%2520measure_type%3Aestimation_procedure&type=measure>`_.
+#
+# Task ``233`` is a simple task using the holdout estimation procedure and therefore has only a
+# single repeat, a single fold and a single sample size:
+
+print(n_repeats, n_folds, n_samples)
```

Comment:
use logging instead of print (?) also, little bit more verbose would be good: 
```
logging.info('dataset %s, %d repeats, %d folds, %d samples' % (task.get_dataset.name, n_repeats, n_folds, n_samples))
```

Reply:
I made this more verbose.